backend/official_plugins/tracim_backend_video_alert/__init__.py

```diff
@@ -0,0 +1,180 @@
+from pluggy import PluginManager
+from pyramid.config import Configurator
+
+from tracim_backend import sliced_dict
+from tracim_backend.lib.core.content import ContentApi
+from tracim_backend.lib.core.plugins import hookimpl
+from tracim_backend.lib.core.user import UserApi
+from tracim_backend.lib.core.workspace import WorkspaceApi
+from tracim_backend.lib.utils.logger import logger
+from tracim_backend.lib.utils.request import TracimContext
+from tracim_backend.lib.utils.utils import string_to_unique_item_list
+from tracim_backend.models.data import Content
+
+# Envoi de la mention
+
+# --- CONFIG ---
+
+MENTION_FIELD = "mention"
+
+config = {
```

Comment:
Maybe rename the whole object "default_config" ? since it is only set here as a default config and a demonstration of what can be set and how it can be set.

Reply:
fixed